dashboards-reports/common/index.ts

```diff
@@ -39,3 +39,17 @@ export const OPENSEARCH_REPORTS_API = {
   LIST_REPORT_DEFINITIONS: `${BASE_REPORTS_URI}/definitions`,
   POLL_REPORT_INSTANCE: `${BASE_REPORTS_URI}/poll_instance`,
 };
+
+const NOTIFICATIONS_API_PREFIX = '/api/reporting_notifications';
+export const NOTIFICATIONS_DASHBOARDS_API = Object.freeze({
```

Comment:
this should be reporting_notifications_api? since it's pointing to `/api/reporting_notifications` inside reporting rather than notifications dashboards plugin

Reply:
Sure, will rename

Reply:
Cannot respond to your other comment for some reason but will add a more verbose error message for test message failures